x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/test_query_row.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiButtonEmpty, EuiFormRow, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useTestQuery } from './use_test_query';
+
+export function TestQueryRow({
+  fetch,
+  hasValidationErrors,
+}: {
+  fetch: () => Promise<{ nrOfDocs: number; timeWindow: string }>;
+  hasValidationErrors: boolean;
+}) {
+  const { onTestQuery, testQueryResult, testQueryError } = useTestQuery(fetch);
+
+  return (
+    <>
+      <EuiFormRow>
+        <EuiButtonEmpty
```

Comment:
![D698E1B8-2F3D-4F94-BE49-5D1C9D78AE90_4_5005_c](https://user-images.githubusercontent.com/39378793/169800613-b2667a6c-fc7e-4571-91b6-cf2b2b52fb36.jpeg)

I think we should add padding for the `Test query` button

Reply:
I think it's done this way, because the left side of the icon should be aligned to the upper block. which looks fine, unless it's focused, then it looks like in the screen. I would suggest to redesign this is in a follow up PR. Maybe a 'filled' button with showing the results in a popover would be an option? FYI @andreadelrio